inputfiles/addedTypes.json

```diff
@@ -2569,6 +2569,28 @@
                         }
                     }
                 }
+            },
+            "WorkerOptions": {
+                "name": "WorkerOptions",
+                "members": {
+                    "member": {
+                        "type": {
+                            "name": "type",
+                            "override-type": "WorkerType",
```

Comment:
Please use `type` instead of `override-type`, as it's only for overriding existing types. Same for line 2584 and 2589.